src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
another console error when moving from doublet filter step to data integration:

<img width="923" alt="Screenshot 2021-02-10 at 10 13 01" src="https://user-images.githubusercontent.com/65414651/107489133-908fce80-6b88-11eb-8816-bce69d5d5fd9.png">

Can you fix it too?

Reply:
Fixed in next commit